docs/api.md

```diff
@@ -276,6 +276,23 @@ If present, signs OS X target apps when the host platform is OS X and XCode is i
 - `entitlements` (*String*): The path to the 'parent' entitlements.
 - `entitlements-inherit` (*String*): The path to the 'child' entitlements.
 
+##### `protocol`
+
+*String*
+
+The URL protocol scheme to associate the app with. Specifying `myapp` would
+cause URLs such as `myapp://path` to be opened with the app. This option can be
+specified multiple times to register multiple protocol schemes with the app.
+Maps to the `CFBundleURLSchemes` metadata property. This argument requires
+a corresponding `--protocol-name` argument.
```

Comment:
This is currently written more CLI-centric than API-centric (although it's a little too verbose for `usage.txt`). `s/argument/option/g` and remove the flag prefix from `protocol-name`.